components/shared/Card.js

```diff
@@ -16,11 +16,15 @@ import styles from './Card.module.css'
  * @param {String} title - is the title of the card
  * @param {String} img - the link to the image
  * @param {String} btnText - inner text for button
+ * @param {String} link - link to where the button leads
+ * @param {Boolean} disabled - true if button should be disabled
  * @param {React.Component} children - is the paragraph text
  */
-function AppCard({ title, img, children, btnText }) {
+function AppCard({ title, img, btnText, disabled, link, children }) {
   btnText = btnText ? btnText : 'READ MORE'
   title = title ? <Card.Title>{title}</Card.Title> : null
+  link = link ? link : '#'
+  disabled = disabled ? true : false
```

Comment:
lol? what's the point of this

Reply:
whoops. only wanted to highlight line 27

Reply:
prop to add a special class to the button to show it's disabled